src/clj/game/cards/agendas.clj

```diff
@@ -515,11 +515,12 @@
 
    "Personality Profiles"
    (let [pp {:req (req (pos? (count (:hand runner))))
-             :msg "force the Runner to trash 1 card from their Grip at random"
-             :effect (effect (trash (first (shuffle (:hand runner)))))}]
+             :effect (effect (trash (first (shuffle (:hand runner)))))
+             :msg (msg "force the Runner to trash 1 card from their Grip at random. Trashes: "
+                       (:title (first (:discard runner))))}]
      {:events {:searched-stack pp
                :runner-install (assoc pp :req (req (and (some #{:discard} (:previous-zone target))
-                                                        (pos? (count (:hand runner))))))}})
+                                                     (pos? (count (:hand runner))))))}})
```

Comment:
This indentation is actually part of the style. We follow https://github.com/bbatsov/clojure-style-guide (or at least try to).